### PR TITLE
[v4] export `style-prefixed.css` with prefixed Tailwind CSS v4 layers for projects which uses Tailwind CSS v3

### PR DESCRIPTION
**Pull request by dimaMachina**

### CODE REVIEW

**Review by hasparus — APPROVED** [review]
nice one

File: packages/nextra-theme-blog/tsconfig.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es2018",
+    "target": "es2021",
```
Comment (hasparus) [inline packages/nextra-theme-blog/tsconfig.json:3]: nice

File: .changeset/gorgeous-forks-hunt.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+"nextra-theme-docs": minor
+---
+
+export `style-prefixed.css` with prefixed Tailwind CSS v4 layers
```
Comment (dimaMachina) [inline .changeset/gorgeous-forks-hunt.md:5]:
```suggestion
export a `style-prefixed.css` file with Tailwind CSS v4 prefixed layers for projects using Tailwind CSS v3.
```

**Comment by marklundin** [comment]
Just to clarify @dimaMachina, does this PR mean we can use tailwind v3 in Nextra v4 by using  `import "nextra-theme-docs/style.css"` to import styles?

**Comment by dimaMachina** [comment]
if you use tailwind css v3 in your project import `style-prefixed.css` instead

**Comment by marklundin** [comment]
Thanks @dimaMachina. Is this in the app-router-34 pre-release, or do we need do wait for the next one?

**Comment by dimaMachina** [comment]
It’s already in last alpha release

**Pull request merged**